### DISCUSSION TITLE
False negative returned for some control plane servers

### DISCUSSION THREAD

**Discussion by cdwbrad** [aquasecurity/trivy-operator#1725] [Help & Support]
I have a Rancher RKE2 cluster with three (3) control plane servers.  By default the `--anonymous-auth` flag is set to false (on all three servers).  However, trivy-operator's CIS compliance report fails two of the control plane servers.  I've verified that the flag is set on all three servers.

So is this type of behavior expected?  Or is there a trivy-operator configuration that I should be tweaking?

**Comment by chen-keinan** [comment]
Issue should be resolved with trivy-operator v0.18.0-rc, please check it
out.
